solutions/bronze/usaco-1251.mdx

```diff
@@ -0,0 +1,64 @@
+---
+id: usaco-1251
+source: USACO Bronze 2022 December
+title: Cow College
+author: Chu Minjae
+---
+[Official Analysis (C++, Java, Python)](http://www.usaco.org/current/data/sol_prob1_bronze_dec22.html)
+
+## Explanation
+
+We can solve this problem by sorting the maximum tuition rates!
+
+There are two reasons for doing this:
+
+1. We need to find the smallest optimal tuition, so by searching the value from the smallest to the largest, we can get the smallest optimal tuition.
+
+2. We can easily calculate the current tuition from the current index.
+
+As we iterate through the list, we can calculate the total tuition at an index $i$ with the equation `maxTuition[i] * (totalNumberOfCows - i)`.
```

Comment:
use latex instead of inline code
also, what's `maxTuition`? use variables that are described in the problem